### PR TITLE
Create DaVinci Formulary files for testing

### PR DESCRIPTION
**Pull request by moiradillon12**
## Description
Create test files to load  search parameters and profiles to meet DaVinci Formulary requirements

## Testing
Touchstone

## FHIR Team Checklist
- [x] **Update the title** of the PR to be succinct and less than 50 characters
- [x] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [x] Tag the PR with **Azure API for FHIR** if this will release to the managed service
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

**Comment by LTA-Thinking** [comment]
Are the C4BB changes just changing the name of the folder? For some reason GitHub isn't showing them as file moves, but as new files.

File: docs/rest/DaVinciFormulary/DaVinci.http [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+content-type: {{contentType}}
+
+< ./Profiles/DavinciCoveragePlan.json
+
+### Davinci Formulary Drug
+POST {{fhirurl}}/StructureDefinition
+Authorization: Bearer {{token}}
+content-type: {{contentType}}
+
+< ./Profiles/DavinciFormularyDrug.json
```
[the remark this answers is not in the document]
Comment (CaitlinV39) [reply docs/rest/DaVinciFormulary/DaVinci.http:86]: @moiradillon12 - Maybe we should add this to both this one and the C4BB?

Get {{fhirurl}}/StructureDefinitions

**Comment by CaitlinV39** [comment]
> Are the C4BB changes just changing the name of the folder? For some reason GitHub isn't showing them as file moves, but as new files.

@LTA-Thinking - I saw the same thing. We're trying to chance the name from CB44 to C4BB. Do you think we can just delete the C4BB?

**Comment by LTA-Thinking** [comment]
The C4BB or the CB44? I don't see any harm in deleting the ones with the wrong name, we don't want duplicates in the code base.

**Comment by moiradillon12** [comment]
@CaitlinV39 @LTA-Thinking my last commit fixed folder name

**Comment by moiradillon12** [comment]
@CaitlinV39 @LTA-Thinking and just added in a check that profiles are there

**Review by CaitlinV39 — APPROVED** [review]

**Pull request merged**